packages/wp-source/src/libraries/handlers/__tests__/__snapshots__/author.tests.ts.snap

```diff
@@ -195,8 +197,10 @@ Object {
       "isFetching": false,
       "isReady": true,
       "items": Array [],
-      "total": NaN,
-      "totalPages": NaN,
+      "next": undefined,
```

Comment:
so, prev and next can be `undefined` if there is no prev or next page.

Reply:
We don't have a particular guideline for this although so far Frontity is more aimed to "missing properties" instead of "undefined properties" o "null properties".